### DISCUSSION TITLE
[RFC] Frontend Tech Stack

### DISCUSSION THREAD

**Discussion by elianiva** [TokoBapak/.github#10] [Frontend Team Posts]
Frontendnya mau pake apa aja?

Possible candidates:
- React, ya.. karena udah mainstream
- Next, same reason
- Svelte, biar rada bliding ej
- Solid, max bliding ej
- yg lain? suggest di reply

UI Library:
- Mantine, kalo pake ini harus pake React
- Tailwind, bisa pake component macem flowbite atau daisy ui atau apalah biar cepet
- Plain CSS / SCSS
- Chakra UI, harus pake React lagi
- yg lain? suggest di reply

Kalo udah fix mau pake apa, biar bisa langsung scaffold reponya

kasih thumbs up kalo setuju di masing masing reply

**Comment by elianiva** [comment]
React

**Comment by elianiva** [comment]
Next

**Comment by elianiva** [comment]
Svelte

**Comment by elianiva** [comment]
Solid

**Comment by elianiva** [comment]
Mantine

**Comment by elianiva** [comment]
Tailwind

**Comment by elianiva** [comment]
Plain CSS / SCSS

**Comment by elianiva** [comment]
Chakra UI

**Comment by zuramai** [comment]
UnoCSS

**Comment by zuramai** [comment]
Nuxt 3 baru release stable

**Comment by zuramai** [comment]
Astro, menyatukan semua framework

**Reply by herbras** [reply]
ini mantap

**Comment by iamyuu** [comment]
Remix

**Reply by herbras** [reply]
kandidat apik

**Comment by iamyuu** [comment]
Qwik
